src/DynamoApplications/StartupUtils.cs

```diff
@@ -224,8 +226,8 @@ public static CommandLineArguments Parse(string[] args)
                 .Add("pi=|PI=|parentId", "Identify Dynamo host analytics parent id", pi => parentId = pi)
                 .Add("da|DA|disableAnalytics", "Disables analytics in Dynamo for the process liftime", da => disableAnalytics = da != null)
                 .Add("cr=|CR=|cerLocation", "Specify the crash error report tool location on disk ", cr => cerLocation = cr)
-                .Add("eng=|ENG=|Eng=|engine=", "true if using the old DesignScript engine, false if using .NET execution of graphs", eng => dsExecutionEngine = !(eng != null && eng.ToLower() == "false"))
-                ;
+                .Add("eng=|ENG=|Eng=|engine=", "True if using the old DesignScript engine, false if using .NET execution of graphs", eng => dsExecutionEngine = !(eng != null && eng.ToLower() == "false"))
+                .Add("rm=|RM=|runMode=", "0 for Compile and Execute mode, 1 for compile-only mode, 2 for execution-only mode for running new engine", rm => msilRunMode = int.Parse(rm) == 0 ? DynamoModel.RunMode.CompileAndExecute : int.Parse(rm) == 1 ? DynamoModel.RunMode.CompileOnly : DynamoModel.RunMode.ExecuteOnly);
```

Comment:
instead of just parsing the string to an int... what about also trying to parse it as an enum directly?
It could be nice for users who were looking just at the client code that invokes the CLI.

`DynamoCLI.exe -eng false -rm 0` ... I won't know what this is doing without using the help docs or finding this source code - but 
`DynamoCLI.exe -eng false -rm CompileAndExecute` 
gives me a fighting chance.

Reply:
also this help doc should probably mention that it's only valid when `eng` is false.

Reply:
> 

I was thinking about this and I felt that CLI users would need to type the exact strings for this to work, e.g. "compileandexecute", etc. I can change it, however, as it improves readability in exchange for verbosity.